tests/CMakeLists.txt

```diff
@@ -74,7 +74,6 @@ set(ARANGODB_TESTS_SOURCES
   Aql/SortLimit-test.cpp
   Aql/SpliceSubqueryOptimizerRuleTest.cpp
   Aql/SplicedSubqueryIntegrationTest.cpp
-  Aql/SubqueryExecutorTest.cpp
```

Comment:
mmmh, why does this fix remove the `SubqueryExecutorTest` and its 500 lines of test code?
Is that intentional? If so, why?

Reply:
The test was not doing anything (most of the test code was `#if 0`d out), and rather than pretend that such a test is there, I decided to delete it.

We have an internal planning ticket to implement tests for this executor.